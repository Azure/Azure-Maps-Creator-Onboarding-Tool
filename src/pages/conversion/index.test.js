import { render } from '@testing-library/react';

import Conversion from './index';

const mockNavigate = jest.fn();

jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
  useNavigate: () => mockNavigate,
}));

describe('Conversion', () => {
  it('should render component', () => {
    const view = render(<Conversion />);
    expect(view).toMatchSnapshot();
  });
});