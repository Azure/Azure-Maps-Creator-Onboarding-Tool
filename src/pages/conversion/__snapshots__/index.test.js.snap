// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Conversion should render component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .emotion-0 {
  position: fixed;
  top: 5.5rem;
  height: calc(100% - 5.5rem - 2px);
  width: calc(100% - 2.5rem);
  left: 1.25rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 0.813rem;
  border-top: 1px solid #E1DFDD;
}

.emotion-1 {
  width: 15rem;
  border-right: 1px solid #E1DFDD;
}

@media (max-width: 600px) {
  .emotion-1 {
    width: 9rem;
  }
}

.emotion-2 {
  height: 2.75rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  padding: 0 0.5rem;
  cursor: pointer;
  border: none;
  width: 100%;
  background-color: #E1DFDD;
  font-weight: 700;
}

.emotion-2:hover {
  color: #0078d4;
  background-color: #F3F2F1;
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-4 {
  margin-right: 0.5rem;
}

.emotion-5 {
  color: #605e5c;
  font-size: 0.6875rem;
}

.emotion-6 {
  height: 2.75rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  padding: 0 0.5rem;
  cursor: pointer;
  border: none;
  width: 100%;
}

.emotion-6:hover {
  color: #0078d4;
  background-color: #F3F2F1;
}

.emotion-18 {
  padding: 0 1rem;
  overflow: auto;
  width: calc(100% - 18rem);
}

@media (max-width: 600px) {
  .emotion-18 {
    width: calc(100% - 9rem);
  }
}

.emotion-19 {
  overflow-x: auto;
}

<body>
    <div>
      <div
        class="emotion-0"
      >
        <div
          class="emotion-1"
        >
          <button
            aria-label="select.upload.step"
            class="emotion-2"
          >
            <div
              class="emotion-3"
            >
              <i
                aria-label="package.conversion"
                class="emotion-4 root-111"
                data-icon-name="StatusCircleRing"
                role="img"
              >
                
              </i>
              package.upload
            </div>
            <span
              class="emotion-5"
            />
          </button>
          <button
            aria-label="select.conversion.step"
            class="emotion-6"
          >
            <div
              class="emotion-3"
            >
              <i
                aria-label="package.conversion"
                class="emotion-4 root-111"
                data-icon-name="StatusCircleRing"
                role="img"
              >
                
              </i>
              package.conversion
            </div>
            <span
              class="emotion-5"
            />
          </button>
          <button
            aria-label="select.dataset.step"
            class="emotion-6"
          >
            <div
              class="emotion-3"
            >
              <i
                aria-label="package.conversion"
                class="emotion-4 root-111"
                data-icon-name="StatusCircleRing"
                role="img"
              >
                
              </i>
              dataset.creation
            </div>
            <span
              class="emotion-5"
            />
          </button>
          <button
            aria-label="select.tileset.step"
            class="emotion-6"
          >
            <div
              class="emotion-3"
            >
              <i
                aria-label="package.conversion"
                class="emotion-4 root-111"
                data-icon-name="StatusCircleRing"
                role="img"
              >
                
              </i>
              tileset.creation
            </div>
            <span
              class="emotion-5"
            />
          </button>
        </div>
        <div
          class="emotion-18"
        >
          <div>
            <div>
              <h3>
                meta.data
              </h3>
              <div>
                operationId: 
                N/A
              </div>
              <div>
                udid: 
                N/A
              </div>
            </div>
            <div>
              <h3>
                operation.log
              </h3>
              <pre
                class="emotion-19"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": .emotion-0 {
  position: fixed;
  top: 5.5rem;
  height: calc(100% - 5.5rem - 2px);
  width: calc(100% - 2.5rem);
  left: 1.25rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 0.813rem;
  border-top: 1px solid #E1DFDD;
}

.emotion-1 {
  width: 15rem;
  border-right: 1px solid #E1DFDD;
}

@media (max-width: 600px) {
  .emotion-1 {
    width: 9rem;
  }
}

.emotion-2 {
  height: 2.75rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  padding: 0 0.5rem;
  cursor: pointer;
  border: none;
  width: 100%;
  background-color: #E1DFDD;
  font-weight: 700;
}

.emotion-2:hover {
  color: #0078d4;
  background-color: #F3F2F1;
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-4 {
  margin-right: 0.5rem;
}

.emotion-5 {
  color: #605e5c;
  font-size: 0.6875rem;
}

.emotion-6 {
  height: 2.75rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  padding: 0 0.5rem;
  cursor: pointer;
  border: none;
  width: 100%;
}

.emotion-6:hover {
  color: #0078d4;
  background-color: #F3F2F1;
}

.emotion-18 {
  padding: 0 1rem;
  overflow: auto;
  width: calc(100% - 18rem);
}

@media (max-width: 600px) {
  .emotion-18 {
    width: calc(100% - 9rem);
  }
}

.emotion-19 {
  overflow-x: auto;
}

<div>
    <div
      class="emotion-0"
    >
      <div
        class="emotion-1"
      >
        <button
          aria-label="select.upload.step"
          class="emotion-2"
        >
          <div
            class="emotion-3"
          >
            <i
              aria-label="package.conversion"
              class="emotion-4 root-111"
              data-icon-name="StatusCircleRing"
              role="img"
            >
              
            </i>
            package.upload
          </div>
          <span
            class="emotion-5"
          />
        </button>
        <button
          aria-label="select.conversion.step"
          class="emotion-6"
        >
          <div
            class="emotion-3"
          >
            <i
              aria-label="package.conversion"
              class="emotion-4 root-111"
              data-icon-name="StatusCircleRing"
              role="img"
            >
              
            </i>
            package.conversion
          </div>
          <span
            class="emotion-5"
          />
        </button>
        <button
          aria-label="select.dataset.step"
          class="emotion-6"
        >
          <div
            class="emotion-3"
          >
            <i
              aria-label="package.conversion"
              class="emotion-4 root-111"
              data-icon-name="StatusCircleRing"
              role="img"
            >
              
            </i>
            dataset.creation
          </div>
          <span
            class="emotion-5"
          />
        </button>
        <button
          aria-label="select.tileset.step"
          class="emotion-6"
        >
          <div
            class="emotion-3"
          >
            <i
              aria-label="package.conversion"
              class="emotion-4 root-111"
              data-icon-name="StatusCircleRing"
              role="img"
            >
              
            </i>
            tileset.creation
          </div>
          <span
            class="emotion-5"
          />
        </button>
      </div>
      <div
        class="emotion-18"
      >
        <div>
          <div>
            <h3>
              meta.data
            </h3>
            <div>
              operationId: 
              N/A
            </div>
            <div>
              udid: 
              N/A
            </div>
          </div>
          <div>
            <h3>
              operation.log
            </h3>
            <pre
              class="emotion-19"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
